.github/workflows/load-supp-content.yml

```diff
@@ -0,0 +1,29 @@
+name: Load Supplementary Content
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  load:
+    strategy:
+      max-parallel: 1
+      matrix:
+        environment: ["dev", "val", "prod"]
+    environment:
+      name: ${{ matrix.environment }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: false
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16' # The Go version to download (if necessary) and use.
+      - run: make supplementary_content
```

Comment:
This won't populate the `guidance` folder in a place where the serverless function can reach it. It will build it in the github action, not in aws.